api/blogs.js

```diff
@@ -1,6 +1,12 @@
 const http = require('../http');
 const BLOGS_API_PATH = 'blogs/v3';
 
+/**
+ * @deprecated
+ * Use the corresponding export from local-dev-lib
```

Comment:
Do you think it would be beneficial to refer to it by it's `npm` package name `@hubspot/local-dev-lib`?

Reply:
Addressed in https://github.com/HubSpot/cli-lib/pull/52/commits/2ef36ad5041355ffbf0056966a50acc8765fba9b